src/core_modules/capture-core/components/DataEntries/Enrollment/actions/open.actionBatchs.js

```diff
@@ -101,13 +108,19 @@ export const openDataEntryForNewEnrollmentBatchAsync = async ({
         formFoundation,
     });
 
+    const effectsWithValidations = await validateAssignEffects({
```

Comment:
If I'm not mistaken, we are validating all the form values in the formbuilder when opening the form, so it would probably be best to skip the assign validations here (and other places where we are opening forms)

Reply:
I skipped the assign validations when opening most of the forms. However, there are two scenarios where running the validations appears necessary: when opening the new event forms for both tracker and single events. In both cases, it seems that the execution of the [rules engine is intentionally delayed](https://github.com/dhis2/capture-app/blob/DHIS2-17854/src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/useLifecycle.js#L44-L105) until after the data entry is opened. This behavior was introduced in [TECH-635](https://dhis2.atlassian.net/browse/TECH-635).